xline/src/server/xline_server.rs

```diff
@@ -28,8 +30,6 @@ use crate::storage::KvStore;
 pub struct XlineServer {
     /// Server name
     name: String,
-    /// Address of server
-    addr: SocketAddr,
```

Comment:
Ok to remove for now. May need to store this information in the future.

Reply:
Yes. Let's add it back later if needed.